tests/integration/component.go

```diff
@@ -712,4 +713,72 @@ func componentTests(args ...string) {
 			Expect(stdError).To(ContainSubstring("quantities must match the regular expression"))
 		})
 	})
+
+	Context("Creating component using symlink", func() {
+		JustBeforeEach(func() {
+			project = helper.CreateRandProject()
+			originalDir = helper.Getwd()
+			helper.Chdir(filepath.Dir(context))
+		})
+		JustAfterEach(func() {
+			// remove the symlinks
+			for _, symLinkPath := range symLinkPaths {
```

Comment:
Race condition will be hit In parallel run in case both the spec runs in same time. Instead of string slice use simple string type.